python/lsst/ctrl/bps/wms/panda/edgenode/cmd_line_decoder.py

```diff
@@ -7,5 +7,9 @@
 import sys
 import binascii
 cmdline = str(binascii.unhexlify(sys.argv[1]).decode())
-cmdline = cmdline.replace("${IN/L}", sys.argv[2])
+dataparams = sys.argv[2].split(":")
```

Comment:
Do dataparams come from the pseudo input filename?  If yes, I didn't follow why ":" here, but "+" in get_pseudo_input_file_name().  Could use a couple short comments to make code easier to understand.

Reply:
Fixed